### PR TITLE
Sets bpp to 4 in X11 Server example

### PR DESCRIPTION
**Pull request by nicmorais**
Fixes: #546 

This pull request fixes glitchy screen when using tight encoding.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -562,6 +562,7 @@ set(LIBVNCSERVER_EXAMPLES
     simple15
     storepasswd
     vncev
+
```
Comment (bk138) [inline CMakeLists.txt:565]: Please don't add unrelated changes :-)

File: examples/server/x11.c [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 void dirty_copy(rfbScreenInfoPtr rfbScreen, const uint8_t* data, int width, int height, int nbytes);
 void convert_bgrx_to_rgb(const uint8_t* in, uint16_t width, uint16_t height, uint8_t* buff);
-void get_window_size(xcb_connection_t* conn, xcb_window_t window, uint16_t* width, uint16_t* height);
+void get_window_size(xcb_connection_t* conn, xcb_window_t window, int16_t* width, int16_t* height);
```
Comment (bk138) [inline examples/server/x11.c:14]: Is this really needed? If not, please leave it out.

File: examples/server/x11.c [hunk 3]
```diff
@@ -136,8 +137,8 @@ void get_window_size(xcb_connection_t* conn, xcb_window_t window, uint16_t* widt
 
 void get_window_image(xcb_connection_t* conn, xcb_window_t window, uint8_t* buff)
 {
-    uint16_t width = 0;
-    uint16_t height = 0;
+    int16_t width = 0;
```
Comment (bk138) [inline examples/server/x11.c:140]: Type change needed here?

**Review by bk138 — CHANGES_REQUESTED** [review]
Thanks so much! Some minor questions and one change request.

File: examples/server/x11.c (hunk 2 above)
Comment (nicmorais) [reply examples/server/x11.c]: Oops! Not needed...

File: examples/server/x11.c (hunk 3 above)
Comment (nicmorais) [reply examples/server/x11.c]: Also not needed, I think I need more coffee

**Comment by bk138** [comment]
@nicmorais lookin good, I'll do a runtime test later today and then this can be merged, I think.
